config/application.yml.default

```diff
@@ -20,6 +20,9 @@ aamva_cert_enabled: true
 aamva_supported_jurisdictions: '["AL","AR","AZ","CO","CT","DC","DE","FL","GA","HI","IA","ID","IL","IN","KS","KY","MA","MD","ME","MI","MO","MS","MT","NC","ND","NE","NJ","NM","NV","OH","OR","PA","RI","SC","SD","TN","TX","VA","VT","WA","WI","WV","WY"]'
 aamva_verification_request_timeout: 5.0
 aamva_verification_url: https://example.org:12345/verification/url
+ab_testing_idv_ten_digit_otp_enabled: false
+ab_testing_idv_ten_digit_otp_percent: 0
+ab_testing_idv_ten_digit_otp_length: 10
```

Comment:
how come this is a config? seems like we could reasonably hardcode 10 here seeing as its the "ten_digit_test"

Reply:
Done in [c75dc8b](https://github.com/18F/identity-idp/pull/10480/commits/c75dc8b1b7ddcbda1340e5f9c52644cd62943386)